### PR TITLE
correct misleading text in podman

### LINKED ISSUE #231: podman init needs updating
`The simplest Skupper site allows you to link to other sites, but does not support linking to the current site.`
is incorrect in context of `skupper init`

### PR DESCRIPTION
**Pull request by pwright**
closes #231

### CODE REVIEW

File: cli/podman.adoc [hunk 1]
```diff
@@ -104,16 +104,30 @@ podman
...
+----
 It is recommended to enable lingering for <username>, otherwise Skupper may not start on boot.
 Skupper is now installed for user '<username>'.  Use 'skupper status' to get more information.
 ----
 
+If you want to limit link creation to be a specific interface, for example to an interface associated with the IP address 192.168.1.10:
+
+[source, bash]
+----
+$ skupper init --ingress-host 192.168.1.10
```
Comment (fgiorgetti) [inline cli/podman.adoc:127]: As an alternative, if no "ingress-host" is provided, skupper tries to include all local IP addresses
as valid ingress hosts, so you could also use `--ingress-host <IP/Hostname>` along with `skupper token create`.

Comment (pwright) [reply cli/podman.adoc]: @fgiorgetti  I've updated, hopefully capturing your message

File: cli/podman.adoc [hunk 2]
```diff
@@ -104,16 +104,37 @@ podman
...
+----
+# loginctl enable-linger <username>
+----
+
+By default, `skupper init` tries to include all IP addresses associated with local network interfaces as valid ingress hosts.
+You can use `--ingress-host <IP/Hostname>` to restrict token ingress to a specific network context:
+
+[source, bash]
+----
+$ skupper init --ingress-host my-cloud-vm.example.com
```
Comment (fgiorgetti) [inline cli/podman.adoc:135]: It is better. WDYT about adding a note that `--ingress-host` can also be used when you create a token?
This way, if someone has multiple ingress hosts, they can specify exactly which host will be defined in
the token, when the token is created.

Comment (fgiorgetti) [reply cli/podman.adoc:135]: By the way, this could be handled separately if you find appropriate.

File: cli/podman.adoc [hunk 3]
```diff
@@ -164,14 +185,12 @@ Linking two sites requires a single initial directional connection. However:
 $ skupper token create <filename>
 ----
 
-The `metadata` section of the resulting YAML file shows which interface is available for linking.
-
-To specify the interface, use `ifconfig` to determine the IP address of the interface you want to use.
-You can then specify that IP address when creating the token:
+If you created the site  without specifying an `ingress-host`, the token is valid for all network contexts.
+You can use `--ingress-host <IP/Hostname>` to restrict token ingress to a specific network context:
```
Comment (pwright) [inline cli/podman.adoc:189]: I think these changes make it clearer, wdyt? @fgiorgetti

File: cli/podman.adoc (hunk 2 above)
Comment (pwright) [reply cli/podman.adoc:135]: I think that should be handled in token section, but good point to check that out, I've made changes there and pinged you

**Review by fgiorgetti — APPROVED** [review]
LGTM. Thank you @pwright .

**Pull request merged**